test/sub_examples/fixtures/rapid_scada_post_send.html

```diff
@@ -0,0 +1,182 @@
+<!doctype html>
```

Comment:
Do you really need this file.

Reply:
Dupe of https://github.com/pymodbus-dev/pymodbus/pull/1665#discussion_r1267612329, resolving this thread